prisma/schema.prisma

```diff
@@ -13,21 +13,22 @@ generator client {
 model Transfer {
   id           String         @id @default(auto()) @map("_id") @db.ObjectId
   depositNonce Int            @unique
-  resource     Resource       @relation(fields: [resourceID], references: [resourceId])
-  resourceID   String
-  fromDomain   Domain         @relation(name: "fromDomainRelation", fields: [fromDomainId], references: [domainId])
+  resource     Resource?      @relation(fields: [resourceID], references: [id])
+  resourceID   String?
+  fromDomain   Domain         @relation(name: "fromDomainRelation", fields: [fromDomainId], references: [id])
   fromDomainId String
-  toDomain     Domain         @relation(name: "toDomainRelation", fields: [toDomainId], references: [domainId])
-  toDomainId   String
-  type         String
-  sender       String
-  amount       String
-  destination  String
-  status       TransferStatus @default(pending)
-  fee          Fee?
+  toDomain     Domain?        @relation(name: "toDomainRelation", fields: [toDomainId], references: [id])
+  toDomainId   String?
+  sender       String?
```

Comment:
Also, create an issue to revisit these ?.
Because for sure sender and timestamp and destination shouldn't be nullable

Reply:
done